src/http/output/metadata/MappedMetadataWriter.ts

```diff
@@ -17,7 +18,7 @@ export class MappedMetadataWriter extends MetadataWriter {
 
   public async handle(input: { response: HttpResponse; metadata: RepresentationMetadata }): Promise<void> {
     for (const [ predicate, header ] of this.headerMap) {
```

Comment:
Same comment here as the `LinkRelMetadataWriter`. Also no idea why this is a `[string, string][]`, might be a relic of when Components.js didn't support Record yet.